.gitignore

```diff
@@ -2,7 +2,6 @@
 old
 .idea
 vendor
-composer.lock
```

Comment:
Why remove this one?

Reply:
To generate a release it is necessary for those who install the package to install exactly all the dependencies used to generate the release. If the package does not have composer.lock the newest version of each dependency will be installed and this may cause php-java not to work.

Reply:
I versioned the composer.lock file

https://getcomposer.org/doc/01-basic-usage.md#commit-your-composer-lock-file-to-version-control

Reply:
@vitormattos

Did you see the note? I'm thinking that the php-java is a library.

> Note: For libraries it is not necessary to commit the lock file, see also: Libraries - Lock file.

https://github.com/php-java/php-java/blob/master/composer.json#L4